src/actions/router.js

```diff
@@ -77,7 +77,6 @@ export const goToLastUsedChannel = () => (dispatch, getState) => {
   const channels = channelsSelector(getState())
   const channel = findLastUsedChannel(channels)
   if (channel) dispatch(goToChannel(channel))
-  else dispatch(handleBadChannel())
```

Comment:
Already handled here: https://github.com/ubergrape/grape-web-client/blob/master/src/actions/channel.js#L254

This line in case channel didn't found triggers recursion, because `handleBadChannel ` calls `goToLastUsedChannel ` again.